src/NosCore.Packets/ClientPackets/Instance/WreqPacket.cs

```diff
@@ -0,0 +1,15 @@
+using NosCore.Packets.Attributes;
+using NosCore.Packets.Enumerations;
+
+namespace NosCore.Packets.ClientPackets.Instance
+{
+	[PacketHeader("wreq", Scope.InGame)]
+    public class WreqPacket : PacketBase
+    {
+        [PacketIndex(0)] 
```

Comment:
Value and param are not really helpful names also it is unformated and the test have to be ran and approved